CHANGELOG.md

```diff
@@ -8,6 +8,14 @@
   substituted at parsing time with a string corresponding to the
   location, filename, line or column number associated to the
   magic constant's position.
+* Multiline string literals are now has new line (`\n`) for each line of the
```

Comment:
Looks like you started writing one sentence then went for another but there are still remnants from the old one making it unintelligible (at least that's what happens to me when I produce similar sentences).